internal/pkg/source/source.go

```diff
@@ -26,6 +26,8 @@ import (
 	"golang.org/x/tools/go/ssa"
 )
 
+// A Classifier is used to identify Objects of interest,
+// such as Sources, Sanitizers, and Sinks.
```

Comment:
Out of scope, but this `config` interface really feels like it belongs with `package config`.  Its inclusion here feels vestigial.

Reply:
Agreed. While we're on this topic, I'm not convinced that `Classifier` is needed right now. So far I don't believe it has shielded us from any changes to `config.Config`, and in fact when we changed `config.Config`'s interface we had to change `Classifier` as well. I think we should consider consuming `config.Config` directly.

Reply:
Also a good point.  I think it may had been introduced to accommodate some tests, but may have only been used by the `source_test`s that were removed in my recent refactor.